rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/values.kt

```diff
@@ -24,6 +24,7 @@ abstract class Value {
 
 interface NumberValue {
     fun negate(): Value
+
```

Comment:
I think we should remove this extra blank line

Reply:
done